server.js

```diff
@@ -1,14 +1,38 @@
-const express = require('express');
-const path = require('path');
+require('dotenv').config();
+
+const express = require("express");
+require('express-async-errors');
+const bodyParser = require("body-parser");
+const cors = require("cors");
+
 const app = express();
-const port = process.env.PORT || 8080;
+const PORT = process.env.PORT || 8080;
+
+const { errorHandlerMiddleware } = require('./api/middleware/errorHandlerMiddleware');
+
+const corsOptions = {
+  origin: "http://localhost:3000"
+};
+
+app.use(cors(corsOptions));
 
-app.use(express.static(path.join(__dirname, 'build')));
+app.use(bodyParser.json());
 
-app.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, 'build', 'index.html'));
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const db = require("./api/models");
+db.sequelize.sync({ force: false }).then(() => {
+  console.log('Done');
+});
```

Comment:
Take it out from root file. You can create smth like `database/index.js` and create a function for connection there